### DISCUSSION TITLE
Angular Image path is not valid

### DISCUSSION THREAD

**Discussion by Najmirza** [community/community#52883] [Programming Help]
I was start angular project but i can't access any css and js file path properly . Kindly do me a favour

**Comment by Najmirza** [comment]
Error showing in Chrome console Refused to apply style from 'http://localhost:4200/src/WebContent/home/assets/css/bootstrap.min.css' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Najmirza ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
